MIDIchlorians/MIDIchlorians/Config.swift

```diff
@@ -115,7 +115,7 @@ struct Config {
     static let SoundExt = "wav"
     static let AnimationExt = "json"
     static let SessionExt = "json"
-
+    static let DefaultSessionsName = "DefaultSessions"
```

Comment:
Oh this is just the name of the file, I think I can use a better name. Will merge this in first and change the name later